### PR TITLE
feat: bump polywrap-client version to 0.1.0a33

### PR DESCRIPTION
**Pull request by Niraj-Kamdar**

### CODE REVIEW

File: implementations/py/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# polywrap-ethereum-plugin
+The Ethereum Provider plugin implements the `ethereum-provider-interface` @ [ens/wraps.eth:ethereum-provider@2.0.0](https://app.ens.domains/name/wraps.eth/details) (see [../../interface/polywrap.graphql](../../interface/polywrap.graphql)). It handles Ethereum wallet transaction signatures and sends JSON RPC requests for the Ethereum wrapper.
+
+## Usage
+### 1. Configure Client
+When creating your Polywrap JS client, add the ethereum provider plugin:
```
Comment (cbrzn) [inline implementations/py/README.md:6]:
```suggestion
When creating your Polywrap Python client, add the ethereum wallet plugin:

File: implementations/py/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    default_network="sepolia",
+    signer=account.key if with_signer else None,  # type: ignore
+)
+
+ethreum_provider_interface_uri = Uri.from_str("ens/wraps.eth:ethereum-provider@2.0.0")
+
+client_config = (
+    PolywrapClientConfigBuilder()
+    .set_package(ethereum_provider_plugin_uri, ethereum_provider_plugin(connections=connections))
+    .add_interface_implementations(ethreum_provider_interface_uri, [ethereum_provider_plugin_uri])
```
Comment (cbrzn) [inline implementations/py/README.md:26]: this interface implementation is not needed anymore by the wrap

File: implementations/py/README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+When creating your Polywrap JS client, add the ethereum provider plugin:
+```python
+from polywrap_client import PolywrapClient
+from polywrap_ethereum_provider import ethereum_provider_plugin
+
+ethereum_provider_plugin_uri = Uri.from_str("plugin/ethereum-provider")
+connections = Connections(
+    connections={
+        "mocknet": Connection(provider, None),
+        "sepolia": Connection.from_network(KnownNetwork.sepolia, None)
```
Comment (cbrzn) [inline implementations/py/README.md:15]: this makes me think, we're passing `signer` in `Connection` and `Connections`; do we want this?

File: implementations/py/README.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    .build()
+)
+client = PolywrapClient(client_config)
+```
+
+### 2. Invoke The Ethereum Wrapper
+Invocations to the Ethereum wrapper may trigger sub-invocations to the Ethereum Provider plugin:
+```python
+client.invoke(
+  uri=ethreum_provider_interface_uri,
```
Comment (cbrzn) [inline implementations/py/README.md:37]: nit: this should we `ethereum_wallet_uri`

File: implementations/py/README.md (hunk 3 above)
Comment (Niraj-Kamdar) [reply implementations/py/README.md:15]: Do you mean, why do we allow passing signers in both connection and connections?

Comment (Niraj-Kamdar) [reply implementations/py/README.md:15]: It's just UX, the user may want to use the same key for every connection or may want separate keys for every connection.

File: implementations/py/README.md (hunk 4 above)
Comment (Niraj-Kamdar) [reply implementations/py/README.md:37]: It's currently published as polywrap-ethreum-provider so I think using the provider word wouldn't be confusing.  We can rename the package in the future if we decide to do that.

**Review by dOrgJelli — APPROVED** [review]
LGTM

**Pull request merged**